### PR TITLE
Nap 193 front page ui tuning

### PR DESCRIPTION
**Pull request by rckopteri**

### CODE REVIEW

File: ote/resources/public/css/omat_tyylit.css [hunk 1]
```diff
@@ -120,3 +120,6 @@ div.user-data > div.user-organization {
 div.front-page-add-service {
      border-right: 1px solid grey;
 }
+.front-page-service-table a, .front-page-service-table a:visited {
+    text-decoration: none;
```
Comment (tatut) [inline ote/resources/public/css/omat_tyylit.css]: we should add new styles with stylefy, see ote.style.base for example

File: ote/src/clj/ote/services/transport.clj [hunk 2]
```diff
@@ -33,15 +35,27 @@
...
 
 (defn get-transport-services [db where]
   "Return Vector of transport-services"
   (fetch db ::t-service/transport-service
-                transport-services-passenger-columns
-                where))
+                transport-services-columns
+                where
+                {::specql/order-by ::t-service/type
+                 ::specql/order-direction :desc
```
Comment (tatut) [inline ote/src/clj/ote/services/transport.clj]: put the next two lines of closing parens on this line

File: ote/src/clj/ote/services/transport.clj [hunk 3]
```diff
@@ -60,6 +74,15 @@
...
         (assoc-in [service-key  ::t-service/operation-area] op-area))))
 
+(defn- delete-transport-service
+  "Delete single transport service by id"
+  [db id]
+  ;; Delete operation area first
+  (delete! db ::t-service/operation_area {::t-service/transport-service-id id})
+  ;; Delete service
+  (delete! db ::t-service/transport-service {::t-service/id id})
+  )
```
Comment (tatut) [inline ote/src/clj/ote/services/transport.clj]: Multiple operations are not atomic, wrap these in a transaction.

File: ote/src/clj/ote/services/transport.clj [hunk 4]
```diff
@@ -96,14 +120,18 @@
...
+  (println "DATA: " (pr-str data))
   (let [places (get-in data [::t-service/passenger-transportation ::t-service/operation-area])
         value (-> data
+                  (assoc ::t-service/modified (Timestamp. (.getMillis (time/now))))
                   (update ::t-service/passenger-transportation dissoc ::t-service/operation_area)
                   (update-in [::t-service/passenger-transportation ::t-service/price-classes] fix-price-classes))]
     (jdbc/with-db-transaction [db db]
          (let [new-value (dissoc value :transport-service)
-               transport-service (upsert! db ::t-service/transport-service new-value)]
+               newer-value (if (nil? (get value ::t-service/id))
```
Comment (tatut) [inline ote/src/clj/ote/services/transport.clj]: punny name, you can reuse new-value here as you won't use it later

File: ote/src/clj/ote/services/transport.clj [hunk 5]
```diff
@@ -116,26 +144,30 @@
...
+             (nil? op-area-id)
+             (insert-point-for-transport-service! db
+                                                {:transport-service-id (get transport-service ::t-service/id)
+                                                 :x (first coordinates)
+                                                 :y (second coordinates)})
+             :else (save-point-for-transport-service! db
+                                                      {:id op-area-id
+                                                       :transport-service-id (get transport-service ::t-service/id)
+                                                       :x (first coordinates)
+                                                       :y (second coordinates)})
```
Comment (tatut) [inline ote/src/clj/ote/services/transport.clj]: closing parens

File: ote/src/cljs/ote/app/controller/transport_service.cljs [hunk 6]
```diff
@@ -51,13 +83,13 @@
...
-      (assoc-in [(keyword ::t-service/type) ::t-service/contact-address] (get response ::t-service/contact-address))
-      (assoc-in [(keyword ::t-service/type) ::t-service/contact-phone] (get response  ::t-service/contact-phone))
-      (assoc-in [(keyword ::t-service/type) ::t-service/contact-email] (get response  ::t-service/contact-email))
-      (assoc-in [(keyword ::t-service/type) ::t-service/homepage] (get response  ::t-service/homepage))
       (assoc :page (get response ::t-service/type)
-             :transport-service response)))
-
+             :transport-service response)
+      (move-service-level-keys-to-form
+        (keyword (str "ote.db.transport-service/" (string/replace (str (get response ::t-service/type)) ":" "")))
```
Comment (tatut) [inline ote/src/cljs/ote/app/controller/transport_service.cljs]: dynamically creating keywords is ugly, use `case` switch on the :type

File: ote/src/cljs/ote/app/controller/transport_service.cljs [hunk 7]
```diff
@@ -77,22 +109,18 @@
...
-                (assoc key (get-in service [from key]))
-                (update from dissoc key)))
-          service
-          service-level-keys))
+  DeleteTransportServiceResponse
+  (process-event [{response :response} app]
+    (.log js/console " deletoitiin ")
+
+    ;; Remove deleted service from the app state
+    ;; FIXME: todo
```
Comment (tatut) [inline ote/src/cljs/ote/app/controller/transport_service.cljs]: return `app` here

File: ote/src/cljs/ote/views/front_page.cljs [hunk 8]
```diff
@@ -11,46 +11,80 @@
...
+            [cljs-time.core :as time]
+            [cljs-time.format :as format]
+            [cljs-time.coerce :as coerce]
+            ))
 
-(defn transport-services-listing [e! transport-operator-id services]
-  (when services
+(def custom-formatter (format/formatter "dd.MM.yy H:i:s"))
+
+(defn format-my-time [my-time]
```
Comment (tatut) [inline ote/src/cljs/ote/views/front_page.cljs]: what is "my time"? do you mean local time? Also, this should be `ote.time` namespace

File: ote/src/cljs/ote/views/front_page.cljs [hunk 9]
```diff
@@ -11,46 +11,80 @@
...
+          [ui/table-row-column {:class "hidden-xs hidden-sm "} (format-my-time created)]
+          [ui/table-row-column
+           [ui/icon-button {:on-click #(e! (ts/->ModifyTransportService id))}
+            [ic/content-create]]
+
+           [ui/icon-button {:on-click #(e! (ts/->DeleteTransportService id))}
+            [ic/action-delete]]
+           ]
+          ])
+       services))
```
Comment (tatut) [inline ote/src/cljs/ote/views/front_page.cljs]: closing parens

File: ote/src/clj/ote/services/transport.clj (hunk 2 above)
Comment (rckopteri) [reply ote/src/clj/ote/services/transport.clj]: Fixed

File: ote/src/cljs/ote/app/controller/transport_service.cljs (hunk 7 above)
Comment (rckopteri) [reply ote/src/cljs/ote/app/controller/transport_service.cljs]: Returned

**Comment by rckopteri** [comment]
All requested changes made

**Pull request merged**